package.json

```diff
@@ -57,5 +58,9 @@
         "tslint": "^5.8.0",
         "@types/node": "^8.10.25",
         "@types/mocha": "^2.2.42"
+    },
+    "dependencies": {
+        "semver": "^5.6.0",
```

Comment:
Also include https://www.npmjs.com/package/@types/semver, refer to https://github.com/Microsoft/vscode-azure-iot-edge/blob/0a563be1133cb8a9252a7153c224d6c82b4dda22/package.json#L441

Reply:
fixed